File: fix_pages.py
import re

with open('public/rates.html', 'r') as f:
    rates = f.read()

rates = re.sub(r'<main class="hero">.*?</main>', '', rates, flags=re.DOTALL)
rates = re.sub(r'<section class="how-it-works">.*?</section>', '', rates, flags=re.DOTALL)
rates = re.sub(r'<section class="why-us" id="why-us">.*?</section>', '', rates, flags=re.DOTALL)
rates = re.sub(r'<section class="cta-section">.*?</section>', '', rates, flags=re.DOTALL)
rates = rates.replace('<section class="live-rates"', '<section class="live-rates" style="padding-top: 120px; min-height: 70vh;"')
rates = re.sub(r'<title>.*?</title>', '<title>Live Rates - CryptoPlexTrade</title>', rates)

with open('public/rates.html', 'w') as f:
    f.write(rates)

with open('public/why-us.html', 'r') as f:
    whyus = f.read()

whyus = re.sub(r'<main class="hero">.*?</main>', '', whyus, flags=re.DOTALL)
whyus = re.sub(r'<section class="how-it-works">.*?</section>', '', whyus, flags=re.DOTALL)
whyus = re.sub(r'<section class="live-rates".*?>.*?</section>', '', whyus, flags=re.DOTALL)
whyus = whyus.replace('<section class="why-us"', '<section class="why-us" style="padding-top: 120px;"')
whyus = re.sub(r'<title>.*?</title>', '<title>Features - CryptoPlexTrade</title>', whyus)

with open('public/why-us.html', 'w') as f:
    f.write(whyus)

print("Pages fixed successfully with Python!")

File: fix_pages.js
const fs = require('fs');

// Process rates.html
let ratesHTML = fs.readFileSync('public/rates.html', 'utf8');
ratesHTML = ratesHTML.replace(/<main class="hero">[\s\S]*?<\/main>/, '');
ratesHTML = ratesHTML.replace(/<section class="how-it-works">[\s\S]*?<\/section>/, '');
ratesHTML = ratesHTML.replace(/<section class="why-us" id="why-us">[\s\S]*?<\/section>/, '');
ratesHTML = ratesHTML.replace(/<section class="cta-section">[\s\S]*?<\/section>/, '');
// For layout, we might need a margin on top since hero is gone
ratesHTML = ratesHTML.replace(/<section class="live-rates"/, '<section class="live-rates" style="padding-top: 120px; min-height: 70vh;"');
ratesHTML = ratesHTML.replace(/<title>.*?<\/title>/, '<title>Live Rates - CryptoPlexTrade</title>');
fs.writeFileSync('public/rates.html', ratesHTML);

// Process why-us.html
let whyusHTML = fs.readFileSync('public/why-us.html', 'utf8');
whyusHTML = whyusHTML.replace(/<main class="hero">[\s\S]*?<\/main>/, '');
whyusHTML = whyusHTML.replace(/<section class="how-it-works">[\s\S]*?<\/section>/, '');
whyusHTML = whyusHTML.replace(/<section class="live-rates".*?>[\s\S]*?<\/section>/, '');
// Add padding top
whyusHTML = whyusHTML.replace(/<section class="why-us"/, '<section class="why-us" style="padding-top: 120px;"');
whyusHTML = whyusHTML.replace(/<title>.*?<\/title>/, '<title>Features - CryptoPlexTrade</title>');
fs.writeFileSync('public/why-us.html', whyusHTML);

console.log("Pages fixed!");
